buildkite/bazelci.py

```diff
@@ -1670,8 +1677,13 @@ def main(argv=None):
                                    use_but=args.use_but,
                                    incompatible_flags=args.incompatible_flag)
         elif args.subparsers_name == "runner":
-            configs = fetch_configs(args.http_config, args.file_config)
-            execute_commands(config=configs.get("platforms", None)[args.platform],
+            tasks = get_tasks(args)
+            # there can be multiple tasks for a platform
+            for name in tasks:
```

Comment:
You can use `for name, task in get_tasks(args).items()` instead :)

Reply:
Done.